### DISCUSSION TITLE
issue with downloading how to fix?

### DISCUSSION THREAD

**Discussion by illuminatijattz** [qnblackcat/uYouPlus#1034] [General]
![62735DAC-6F9E-467C-A14D-632FB44B6243](https://user-images.githubusercontent.com/122349532/226622200-abc96a52-9459-4fb5-9b40-73b5f64b768c.png)

**Comment by damiendep** [comment]
I have the same problem!
I think there are "left behinds" from a previous version, or maybe from another fork, that conflict with the download.... and I don't have a solution so far.

**Reply by illuminatijattz** [reply]
i got solution for this if you install uyou using altstore or any other computer method then download and normal sign in works fine but if we use scarlet or any other online method then this issue comes

**Reply by damiendep** [reply]
I'm already using that method (I've developper account + Sideloadly on Windows PC).
I also tried uninstalling completely all instance of Youtube/Cercube/uYou+/... but the problem remains.

**Reply by illuminatijattz** [reply]
try with altstore i used altstore and it works fine i have uyou 18.08.1 ipa

**Reply by damiendep** [reply]
Ok, thanks for the tip.
I'll try that and post back my experience.

**Comment by dayanch96** [comment]
wait for 3.0 ipa release

**Reply by damiendep** [reply]
I'll follow that advice! (Using AltStore poses too many constraints in my case)

**Reply by damiendep** [reply]
3.0 fixed the problem. Thanks for the tip
